helpers.py

```diff
@@ -55,12 +63,12 @@ def _pages_this_day():
     now = datetime.date.today()
 
     # Grab each issue that matches, and create a page_info structure for the
-    # first page of that issue.  For sanity's sake, we only pull 100 max.
+    # first page of that issue.  Grabbing only the set number requested by user
     issues = models.Issue.objects
     issues = issues.filter(date_issued__range=(dt_range_start, dt_range_end))
     issues = issues.filter(date_issued__month = now.month)
     issues = issues.filter(date_issued__day = now.day)
-    for issue in issues[:100]:
+    for issue in issues[:config.NUMBER]:
```

Comment:
Good question.  I just tried similar code in a little online python interpreter but I'll do a little more testing with the django plugin and see if I can determine if it will work.

Reply:
Things that work:
- `0` - displays zero items, no error
- `> 0` - displays up to that amount of items and no error if greater than the total items returned

- `None`- displays at least one item....potentially it would display ALL items??  didn't test much since if that's coming up it probably means things are not configured correctly